backend/controllers/order.js

```diff
@@ -0,0 +1,106 @@
+const ReasonPhrases = require('http-status-codes').ReasonPhrases;
+const StatusCodes = require('http-status-codes').StatusCodes;
+const Order = require('../models/order');
+
+// Handle index actions
+exports.index = function (req, res) {
+  Order.find(function (err, order) {
+    if (err) {
+      console.log(err);
+      res.status(StatusCodes.INTERNAL_SERVER_ERROR).json({
+        message: ReasonPhrases.INTERNAL_SERVER_ERROR,
+      });
+    } else {
+      res.status(StatusCodes.OK).json(order.map(order => {
+        return order;
+      }));
+    }
+  });
+};
+
+// Handle create order actions
+exports.new = function (req, res) {
+  const order = new Order(
+    {
+      product_id: req.body.product_id,
+      buyer_id: req.body.buyer_id,
+      address: {
+        street: req.body.address.street,
+        street_number: req.body.address.street_number,
+        city: req.body.address.city,
+        postal_code: req.body.address.postal_code,
+      },
+      state: req.body.type,
```

Comment:
I would remove this here because the default value already covers that attribute. And it shouldn't be possible to create an already fulfilled order.

Reply:
Agreed